src/transformers/training_args.py

```diff
@@ -210,6 +210,8 @@ class TrainingArguments:
             can harm metric values.
         local_rank (:obj:`int`, `optional`, defaults to -1):
             Rank of the process during distributed training.
+        xpu_backend (:obj:`str`, `optional`, defaults to :obj:`"ccl"`):
+            The backend to use for xpu distributed training. Must be one of :obj:`"mpi"` or :obj:`"ccl"`.
```

Comment:
Should we have this default to None and check when it's set to detect a distributed training on CPU? Right now it's only if we pass `--no_cuda`.

Reply:
Sure.